examples/remarketing/upload_offline_conversion.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example imports offline conversion values for specific clicks.
+
+To get Google Click ID for a click, use the "click_view" resource:
+https://developers.google.com/google-ads/api/fields/latest/click_view.
+To set up a conversion action, run the AddConversionAction.php example.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, conversion_action_id, gcl_id, conversion_time,
+         conversion_value):
+    """Creates a click conversion with a default currency of USD."""
+
+    click_conversion = client.get_type('ClickConversion', version='v2')
+    conversion_action_service = client.get_service('ConversionActionService',
+                                                   version='v2')
+    click_conversion.conversion_action.value = (
+        conversion_action_service.conversion_action_path(
+            customer_id, conversion_action_id)
+    )
+    click_conversion.gclid.value = gcl_id
+    click_conversion.conversion_value.value = float(conversion_value)
+    click_conversion.conversion_date_time.value = conversion_time
+    click_conversion.currency_code.value = 'USD'
+
+    conversion_upload_service = client.get_service('ConversionUploadService',
+                                                   version='v2')
+
+    try:
+        conversion_upload_response = (
+            conversion_upload_service.upload_click_conversions(customer_id,
+                [click_conversion], partial_failure=True)
+        )
+        uploaded_click_conversion = (conversion_upload_response.results[0])
+        print(f'Uploaded conversion that occurred at '
```

Comment:
nit: these two lines do not have to be in the try block

Reply:
I believe that getting the results should be in the try/except because the operation is lazy.

Reply:
SG